### ISSUE TITLE
Error `undefined is not an object (evaluating 't.x')`, probably related to event.touches

### ISSUE THREAD

**Issue by English3000** [manuelstofer/pinchzoom#70]
The error message: 
`TypeError · undefined is not an object (evaluating 't.x') -- in 'getDistance'`

![screen shot 2018-09-28 at 12 00 52 pm](https://user-images.githubusercontent.com/29601050/46229411-61ab9180-c31a-11e8-8a2a-fddc2a34c4e5.png)

`getDistance` has an `undefined` argument.

```js
            getDistance = function (a, b) {
                var x, y;
                x = a.x - b.x; // undefined.x => TypeError
                y = a.y - b.y;
                return Math.sqrt(x * x + y * y);
},
```

It is invoked by `calculateDistance`--which has as its first argument `startTouches`.

`startTouches` is initialized [here](https://github.com/manuelstofer/pinchzoom/blob/master/src/pinch-zoom.js#L737):

```js
var detectGestures = function (el, target) {
        var interaction = null,
            fingers = 0,
            lastTouchStart = null,
            startTouches = null,
```

Looking within the `'touchmove'` [event handler](https://github.com/manuelstofer/pinchzoom/blob/master/src/pinch-zoom.js#L850),

```js
        el.addEventListener('touchmove', function (event) {
            if(target.enabled && !target.isDoubleTap) {
                if (firstMove) {
                    updateInteraction(event);
                    if (interaction) {
                        cancelEvent(event);
                    }
                    startTouches = targetTouches(event.touches);
                } else {
                    switch (interaction) {
                        case 'zoom':
                            target.handleZoom(event, calculateScale(startTouches, targetTouches(event.touches)));
break;
```

`startTouches` is mutated in the `if` block.

Given that the error occurs within `getDistance` and NOT `calculateScale` (in the `else` block):

```js
calculateScale = function (startTouches, endTouches) {
                var startDistance = getDistance(startTouches[0], startTouches[1]),
                    endDistance = getDistance(endTouches[0], endTouches[1]);
                return endDistance / startDistance;
},
```

this means `startTouches` is being properly mutated from `null` to an array, and there must be a case where `event.touches.length === 1` or `0`, meaning `startTouches[1]` or `endTouches[1]` is `undefined`.

**Comment by sandstrom** [comment]
@English3000 Thanks for a good bug report and for starting the investigation!

Is that a Bugsnag report? Looks like it.

If so, how many instances of the error have you got? Can you see any pattern in affected browser/os?

**Comment by sandstrom** [comment]
Should been fixed in #77
